pkg/framework/util.go

```diff
@@ -52,6 +52,9 @@ const (
 	// DefaultContainerImage is the default image for container using
 	DefaultContainerImage string = "busybox:1.28"
 
+	// DefaultContainerVolumeImage is the default image for container with volume using
```

Comment:
Where to use it?

Reply:
Test whether to support the ability to acquire the volumes of the Image when getting image status.